### ISSUE TITLE
Stitching - extending types and passing in multiple ids

### ISSUE THREAD

**Issue by sspikker** [ChilliCream/hotchocolate#3827]
I would like to find out if it is possible to extend a type like the example below where instead of passing in one id (sourceContactId) it is possible to pass in multiple ids. The reason being is that at the moment it will call the contactById method for each addressbook row.

extend type ReportedLossesOutputDto {
    sourceContact: ContactFullOutputDto!
    @delegate(schema: "addressbook", path: "contactById(id: $fields:sourceContactId)")
}

Basically we would like to reduce the amount of times the contactById method gets called. For example if there are 10 address book rows, it currently calls the contactById method 10 times. Is there a way to have maybe a method called "contactsById", which is called once?

**Comment by michaelstaib** [comment]
This is going to be addressed with the august release.

**Comment by sspikker** [comment]
Thank you for the response.

**Comment by heymanitsmematt** [comment]
@michaelstaib  was this included in the August release? I see it was added to backlog after that release.
